app/assets/javascripts/components/amounttable.js

```diff
@@ -0,0 +1,313 @@
+import {toCSVString, downloadData} from "../utils.js";
+
+
+/**
+ * @typedef {Object} AmountTableSettings
+ * @property {string}   el        A CSS selector of the target container
```

Comment:
String is wrong, it is a `d3.selection` what is the convention for those objects?

_[Original comment](https://github.ugent.be/unipept/unipept/pull/625#discussion_r13731) by @beardhatcode on Mon Feb 19 2018 at 15:02._